modules/system/lang/nl/validation.php

```diff
@@ -90,10 +112,42 @@
     |
     | The following language lines are used to swap attribute place-holders
     | with something more reader friendly such as E-Mail Address instead
-    | of 'email'. This simply helps us make messages a little cleaner.
+    | of "email". This simply helps us make messages a little cleaner.
     |
     */
 
-    'attributes' => [],
+    'attributes'           => [
```

Comment:
I'm not sure we want these included in here, they should be up to the plugin authors to add. Ideally we'd like to implement a validation improvement that displays translated field labels instead of the raw attribute names in the FormController. Thoughts?

Reply:
In short: it is work already done that can be seen as fall back measures. It can and should be implemented in the Plugin's itself or in the app itself. But seems a waste to throw it away.
On the other hand, it could lead to wrong assumptions about how to code for OC...
What is the best thing to do?

Reply:
I just removed the extras to keep the core validation messages simple.